scripts/python/lib/utilities.py

```diff
@@ -103,7 +103,8 @@ def has_dhcp_servers(interface):
 
 
 def is_ipaddr(ip):
-    if re.search(r'\A(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}'
+    # not sure of error from flake .. \Z ?
+    if re.search(r'\A(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}'  # noqa: W605
```

Comment:
\A and \Z should be valid (the Python 3.6 RE engine does handle them), but you could probably clear the tox msg by changing the '\A' to '^' and '\Z' to '$'